StortSpelProjekt/Game/src/main.cpp

```diff
@@ -609,6 +610,52 @@ Scene* ShopScene(SceneManager* sm)
     return scene;
 }
 
+void logSchtuff(const std::string& name)
+{
+    Log::Print("%s\n", name.c_str());
+}
+
+Scene* MainMenuScene(SceneManager* sm)
```

Comment:
Why is this scene created in two places?

Reply:
it was a copy pasta from main instead of copy pasta. The function is removed from main.